filament/backend/src/opengl/gl_headers.h

```diff
@@ -98,6 +98,12 @@
 #error "Minimum header version must be OpenGL ES 3.0 or OpenGL 4.1"
 #endif
 
+#if defined(GL_ES_VERSION_3_0)
```

Comment:
I think a better fix would be to only have GLES30_HEADERS -- because truth be told, we can't rely on the 3.1 headers anyways.

That is, just rename GLES31_HEADERS to GLES30_HEADERS and change the test on line 107 to use GL_ES_VERSION_3_0.

Reply:
One issue - I tried that approach first, but had problems with e.g.
https://github.com/google/filament/blob/master/filament/backend/src/opengl/OpenGLDriver.cpp#L583
where those methods aren't provided by the headers

Reply:
I applied your changes - thank you!